src/components/schedule-inputs.tsx

```diff
@@ -499,6 +385,81 @@ function parseTime(input: string): [number | undefined, number | undefined] {
   return [hours, minutes];
 }
 
+// Converts hours and minutes to hh:mm format
+function toClock(hours: number, minutes: number): string {
+  return (
+    (hours < 10 ? '0' + hours : hours) +
+    ':' +
+    (minutes < 10 ? '0' + minutes : minutes)
+  );
+}
+
+type CronTerms = [number, number, number, number, number];
+
+/**
+ * Extracts cron terms and coerces them into an array of numbers. Ranges are
+ * coerced by their first term, e.g. "12-34" is coerced to 12.
```

Comment:
Technically, days of the week like `MON` (1) and `FRI` (5) are also valid cron terms, but this function would coerce them to 0 (Sunday).

Reply:
Agreed, this is expected of the current implementation.